renderer/components/Home/WalletUnlocker.js

```diff
@@ -82,15 +82,15 @@ class WalletUnlocker extends React.Component {
           <>
             <WalletHeader wallet={wallet} />
 
-            <Input
+            <PasswordInput
+              autoComplete="current-password"
```

Comment:
I'd suggest moving this prop into the `PasswordInput` component since I can't imagine a situation where we would not want this. This enabled password managers to interact with the input.

Reply:
@mrfelton what about the recovery seed password input? You wouldn't want it to have this, right?

Reply:
Why not? It would allow you to easily pop up your password manager and use it to generate a password

Reply:
Ok, well actually, I guess not. Perhaps you are right - this should be set per instance as you are now. I wasn't aware of the various different options for this:

![image](https://user-images.githubusercontent.com/200251/63189856-3f6f4f00-c065-11e9-9814-96b32f6a7d82.png)